### PR TITLE
added lists

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
127 0 0 1_5500_Documentation_lists_lists html

### CODE REVIEW

File: Documentation/lists/lists.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+    <footer class="footer">
+        <div class="centered container">
+          <ul class="list-non-bullet">
+            <li class="list-inline">
+              <a
+                class="links"
+                target="_blank"
+                href="https://github.com/Priyanshi-Tiwari"
+                ><i class="fa-brands fa-github" style="font-size: 40px"></i
```
Comment (vanshitalavange) [inline Documentation/lists/lists.html:142]: avoid inline styles, instead increase the specificity and do it

File: Stylesheets/components/lists.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+.list-container{
+    border: 0.1rem solid var(--gray-color);
+    border-radius: 0.5rem;
+    width:35rem;
+    align-items: center;
+    display: flex;
+}
+
+.flex-center{
```
Comment (vanshitalavange) [inline Stylesheets/components/lists.css:30]: suggestion: you can extract this out as an utility

File: Documentation/lists/lists.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="../../Stylesheets/main.css" />
+    <script
+      src="https://kit.fontawesome.com/6026902e47.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <link rel="stylesheet" href="alert.css" />
```
Comment (vanshitalavange) [inline Documentation/lists/lists.html]: why do u need alert.css here?

**Review by vanshitalavange — COMMENTED** [review]
The appearance looks really good, suggested  a few things please look into it

**Review by aatishpandey — APPROVED** [review]
UI looks good.
 Good naming convention for classes.
Overall looks good to merge.
You can work on the suggestions provided above.

File: Documentation/lists/lists.html (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply Documentation/lists/lists.html:142]: okay

File: Stylesheets/components/lists.css (hunk 2 above)
Comment (Priyanshi-Tiwari) [reply Stylesheets/components/lists.css:30]: okay

File: Documentation/lists/lists.html (hunk 3 above)
Comment (Priyanshi-Tiwari) [reply Documentation/lists/lists.html]: thanks for pointing this out

**Pull request merged**